priv/repo/migrations/20170315163823_create_user.exs

```diff
@@ -0,0 +1,17 @@
+defmodule ResearchResource.Repo.Migrations.CreateTable do
+  use Ecto.Migration
+
+  def change do
+    create table :users do
+      add :ttrrid, :string
```

Comment:
ttrrid is the unique id we're using across the different databases (ttrr stands for 'talking therapies research resource')